### PR TITLE
Un-pin 1.12

### PR DESCRIPTION
**Pull request by sanchezl**

### CODE REVIEW

**Comment by sttts** [comment]
These have to wait until https://github.com/openshift/origin/pull/22464 merged and the publishing bot updated apiextensions-apiserver.

/hold

**Comment by sttts** [comment]
Please don't squash the glide.yaml change and the actual bump. This makes the PR unreviewable in Github. Would like to comment on the former, but can't.

File: glide.yaml [hunk 1]
```diff
@@ -1,64 +1,34 @@
...
+  version: kubernetes-1.13.4
 - package: k8s.io/kube-aggregator
-  version: kubernetes-1.12.5
+  version: kubernetes-1.13.4
 - package: k8s.io/code-generator
-  version: kubernetes-1.12.5
+  version: kubernetes-1.13.4
 - package: k8s.io/apiextensions-apiserver
   repo: https://github.com/openshift/kubernetes-apiextensions-apiserver
   version: 36be1826c980fa7ca82d91645e06e3157f159322
```
Comment (deads2k) [inline glide.yaml]: this one is probably broken now.

Comment (sttts) [reply glide.yaml]: It is. We need crd publishing first in origin before we can proceed here.

**Comment by deads2k** [comment]
moral equivalent merged

**Pull request closed without merging**